Add tag detection (#90)
SigmaHQ/sigma-specification: Tags_specification.md

All TLP levels defined by the [FIRST TLP-SIG](https://www.first.org/tlp/) in lower case. Example tag: `tlp.amber`.

### namespace: detection

> Use the detection tag to determine the type of detection targeted by the rule. Example tag: `detection.threat_hunting`.

Reviewer: ```suggestion
Use the detection tag to indicate the type of a rule. Example tag: `detection.threat_hunting`.
```

* dfir
* emerging_threats
* threat_hunting 

## History
* 2023-06-20 Tags V1.1.0

>   * Add detection tags

Reviewer: ```suggestion
  * Add detection namespace
```
